.drone.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
Please drop this file and write it to the drone.yml like we already did for plugins like s3-cache

Reply:
Totally will do, I was following more of the drone-docker path but I like this better since you can group the builds together.